en/docs/setup/setting-up-deployment-pattern-1.md

```diff
@@ -689,10 +625,10 @@ persistence in the
 `          <IS_HOME>/repository/conf/identity/identity.xml         `
 file. For more information on sticky sessions, see [Sticky Sessions with
 Manager
-Nodes](https://docs.wso2.com/display/CLUSTER44x/Sticky+Sessions+with+Manager+Nodes)
+Nodes](../../admin-guide/sticky-sessions-with-manager-nodes)
 . The following is the deployment diagram with the load balancer.
 
-![](attachments/103329476/103329481.png) 
+![deployment with load balancer](..//assets/img/setup/deployment-with-load-balancer.png) 
```

Comment:
../../assets

Reply:
The respective file was removed from the commit as the file is subject to change—it will be reflected in a subsequent PR.